spec.mediawiki

```diff
@@ -448,31 +422,33 @@ listed here. The <code>accountid</code> field is retrieved during registration
 as described in Section 5.1.
 
 {|
-! field     !! type   !! description
+! field       !! type   !! description
 |-
-| accountid || string || client account identifier
+| accountid   || string || client account identifier
 |-
-| timestamp || int    || UTC timestamp generated at the time of order submission of trades requires a fair amount of messaging between the 2 clients, via the DEX trade API.n
+| timestamp   || int    || UTC timestamp generated at the time of order submission of trades requires a fair amount of messaging between the 2 clients, via the DEX trade API.n
 |-
-| tradepair || string || market. trading pair. e.g. DCR/BTC
+| tradepair   || string || market. trading pair. e.g. DCR/BTC
 |-
-| ordertype || string || "limit" for a limit order
+| ordertype   || string || "limit" for a limit order
 |-
-| direction || string || one of "buy" or "sell"
+| direction   || string || one of "buy" or "sell"
 |-
-| ordersize || int    || order size, in atoms (*1e8)
+| ordersize   || int    || order size, in atoms (*1e8)
 |-
-| price     || int    || price, in atoms/Satoshi or similar (*1e8)
+| price       || int    || price, in atoms/Satoshi or similar (*1e8)
 |-
-| utxotxid  || string || transaction hash associated with the utxo
+| timeinforce || string || one of "standing" or "immediate"
 |-
-| utxoindex || int    || the index of the utxo in the transaction outputs
+| utxotxid    || string || transaction hash associated with the utxo
```

Comment:
I'm wondering if we decide to allow, say, up to 4 utxos with combined value of at least the lot size, maybe this should be a `[]string`.  Similarly for the index.  It wouldn't break anything if we only allow 1 utxo, but it would potentially allow the api to support more advanced order composition.  Thoughts?